src/documentation/basic-opts.page

```diff
@@ -30,21 +30,23 @@ Another possible way of sending this information to uniqush: the app send the in
 
 As mentioned in [basic concepts], a subscriber may have more than one delivery point subscribes a service. So several delivery points could subscribe same service under same subscriber. This requires the server side have some mechanism to authenticate the subscriber's identity.
 
-## Push Data to Subscriber
+## Sending Push Notifications to a Subscriber
 
-Once a service has more than one push service provider and more than one subscriber, uniqush can push data on behalf of that service to the apps running on subscribers' devices. Server side only need to provide service name, subscriber name and data. Uniqush will push the data to all active delivery points under that subscriber.
+Once a service has more than one push service provider and more than one subscriber, uniqush can push data on behalf of that service to the apps running on subscribers' devices. Server side only needs to provide the service name, subscriber id and data. Uniqush will push the data to all active delivery points under that subscriber.
```

Comment:
Another thing we should standardize on is capitalizing Uniqush as it's a proper noun